docs/editor-sprite-mask.en-US.md

```diff
@@ -0,0 +1,32 @@
+---
+order: 8
+title: SpriteMask Component
+type: Editor
+---
+
+The sprite mask component is used to mask the [sprite](${docs}editor-sprite-renderer-cn) in the 3D/2D scene. For details, please refer to the [sprite mask component](${docs}sprite-mask- cn).
+
+## Add sprite mask component
+
+When we need to mask a sprite, we first need to create an entity and add the sprite mask component, as follows:
```

Comment:
When we need to mask a sprite, an new entity with the sprite mask component can be created as follows:

more passive words.

Reply:
done